andela-aabdulwahab/bucketlist-django-application
# Move the Develop branch into Master

#### What does this PR do?

Move the develop branch to master
#### Description of Task to be completed?

After complete development of the application, the develop branch is move to master to prepare for first release
#### How should this be manually tested?

Visit the [Application](https://buppli.herokuapp.com) the see it's current state
#### Any background context you want to provide?

This mark the completion of checkpoint three, awaiting trainers review
